### ISSUE TITLE
Bug: 'invnext' and 'invprev' commands not functioning

### ISSUE THREAD

**Issue by FuzzyLeo** [ChaosInitiative/Portal-2-Community-Edition#981]
### Describe the bug

When i tried binding invnext and invprev to buttons then pressed those buttons while i have multiple weapons nothing happened, when it should have switched between those weapons

### Issue Map

Any

### To Reproduce

1. open any map
2. bind buttons to 'invnext' and 'invprev'
3. give yourself weapons
4. try to switch between them using the bound keys

**Comment by FuzzyLeo** [comment]
upon further inspection no command related to weapon switching functions properly apart from 'use [weapon]'

**Comment by Luke18033** [comment]
Yeah, it's because currently weapon switching is entirely tied to the HEV suit, which isn't implemented in Portal 2. (I'm honestly surprised these commands still exist to begin with, I wonder if they even have any code?) Soon weapon switching will be moved into Panorama, so at that point these commands will basically just send events to the UI, if they aren't replaced entirely.

**Comment by FuzzyLeo** [comment]
from reading old chat messages in the discord it seemed like weapon switching using these commands used to work

**Comment by Luke18033** [comment]
It never has as far as I'm aware, I don't know why we would have removed that
